### ISSUE TITLE
LoadError: cannot load such file -- rexml/document in Ruby 3.0.1

### ISSUE THREAD

**Issue by ArturT** [braintree/braintree_ruby#214]
### General information

* SDK/Library version: braintree 4.0.0
* Environment: production
* Language, language version, and OS: Ruby 3.0.1

### Issue description

I've upgraded Ruby from 2.7.1 to Ruby 3.0.1 in my Rails application. I use braintree 4.0.0 gem.

I noticed an error when my API received a webhook from Braintree.

```
LoadError: cannot load such file -- rexml/document
```

for this line:

```ruby
webhook_notification = Braintree::WebhookNotification.parse(params[:bt_signature], params[:bt_payload])
```

I added rexml gem to my Gemfile and it fixed the issue.

I believe in Ruby 3.0.1 the rexml is not part of stdlib so it must be explicitly defined as a dependency in gemspec for braintree gem.

**Comment by scannillo** [comment]
Hi @ArturT - thank you for opening this issue, and for using Braintree Ruby.

Looks like you are correct in that in Ruby 2.6 and 2.7 `rexml` is a default gem, whereas in Ruby 3 `rexml` is a separate bundled gem.

We will keep you posted when we get a fix out for this!

**Comment by scannillo** [comment]
👋  Hello @ArturT - we released a fix for this in[ Braintree Ruby v4.1.0](https://github.com/braintree/braintree_ruby/blob/master/CHANGELOG.md#410). Thank you for opening the issue!
